azure/appservice/main.tf

```diff
@@ -1,79 +1,39 @@
-resource "azurerm_resource_group" "app" {
-  name     = "rg-${var.project}-${var.environment}"
-  location = local.location
-
-  tags = {
-    country     = var.country
-    environment = var.environment
-    squad       = var.squad
-    product     = var.product
-  }
-}
-
 resource "azurerm_app_service_plan" "app" {
   name                = "asp-${var.project}-${var.environment}"
-  location            = azurerm_resource_group.app.location
-  resource_group_name = azurerm_resource_group.app.name
+  location            = local.location
+  resource_group_name = var.resource_group
   kind                = "Linux"
   reserved = true
 
   sku {
     tier = var.plan
     size = var.size
   }
-
-  tags = {
-    country     = var.country
-    environment = var.environment
-    squad       = var.squad
-    product     = var.product
-  }
 }
 
 resource "azurerm_app_service" "app" {
   name                = "as-${var.project}-${var.environment}"
-  location            = azurerm_resource_group.app.location
-  resource_group_name = azurerm_resource_group.app.name
+  location            = local.location
+  resource_group_name = var.resource_group
   app_service_plan_id = azurerm_app_service_plan.app.id
   https_only = true
+
   identity {
     type = "SystemAssigned"
   }
-
-  tags = {
-    country     = var.country
-    environment = var.environment
-    squad       = var.squad
-    product     = var.product
-  }
 }
 
-resource "azurerm_log_analytics_workspace" "apm" {
+resource "azurerm_log_analytics_workspace" "app" {
   name                = "wsp-${var.project}-${var.environment}"
-  location            = azurerm_resource_group.app.location
-  resource_group_name = azurerm_resource_group.app.name  
+  location            = local.location
+  resource_group_name = var.resource_group
   retention_in_days   = 90
-
-  tags = {
-    country     = var.country
-    environment = var.environment
-    squad       = var.squad
-    product     = var.product
-  }
-
 }
 
-resource "azurerm_application_insights" "apm" {  
+resource "azurerm_application_insights" "app" {  
   name                = "apm-${var.project}-${var.environment}"
```

Comment:
This naming convention is not currently what' in use.
e.g. AppIns-PayrollLiveNL-WebApp

Reply:
fixed.